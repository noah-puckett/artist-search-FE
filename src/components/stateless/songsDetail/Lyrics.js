import React from 'react';
import PropTypes from 'prop-types';

function Lyrics({ text }) {
    return (<>
        <p>{text}</p>
    </>
    );
}

Lyrics.propTypes = {
    text: PropTypes.string.isRequired,
};

export default Lyrics;
