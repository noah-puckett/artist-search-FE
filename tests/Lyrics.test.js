import React from 'react';
import { shallow } from 'enzyme';
import Lyrics from '../src/components/stateless/songsDetail/Lyrics';

describe('Lyrics component', () => {
    it('renders Lyrics', () => {
        const wrapper = shallow(<Lyrics lyrics='hello' artist='hello' title='hello' />);
        expect(wrapper).toMatchSnapshot();
    });
});
