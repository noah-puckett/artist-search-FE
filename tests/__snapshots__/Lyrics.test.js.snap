// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Lyrics component renders Lyrics 1`] = `
<Fragment>
  <h1>
    hello
  </h1>
  <h3>
    hello
  </h3>
  <p>
    hello
  </p>
</Fragment>
`;
